### PR TITLE
Metadata

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
Expect some pool listing tests to fail...

**Comment by mulkieran** [comment]
should succeed w/ redirect

**Comment by mulkieran** [comment]
rebased

**Comment by mulkieran** [comment]
Passed against updated D-Bus API...

**Comment by mulkieran** [comment]
expect coverage test to fail...

**Review by drckeefe — APPROVED** [review]

**Review by bgurney-rh — APPROVED** [review]

**Review by bmr-cymru — APPROVED** [review]

File: docs/stratis.txt [hunk 1]
```diff
@@ -41,10 +41,14 @@ pool create [--key-desc <key_desc>] [--clevis <(nbde|tang|tpm2)> [--tang-url <ta
...
      the storage stack but leave all metadata intact.
-pool start <(--uuid <uuid> |--name <name>)> --unlock-method <(keyring | clevis)>::
+pool start [--keyfile-path KEYFILE_PATH | --capture-key] --unlock-method <(any | clevis | keyring)> <(--uuid <uuid> |--name <name>)>::
      Start a pool, specifying the pool by its UUID or by its name. Use the
      --unlock-method option to specify method of unlocking the pool if it is
-     encrypted.
+     encrypted. If an unlock method of any is specified the pool will be
+     unlocked with any facility available that allows the pool to be unlocked.
+     If --keyfile-path or --capture-key is specified, then the pool will use
+     only the passphrase and method specified, so that the pool will not be
```
Comment (jbaublitz) [inline docs/stratis.txt:50]: Maybe "and the keyslot corresponding to the method specified"

**Review by jbaublitz — CHANGES_REQUESTED** [review]
After testing I don't think I have any problems with the code, just one note on documentation that could be a little bit clearer.

**Comment by mulkieran** [comment]
@jbaublitz Ok. I've modified the man page to try to make the pool start rules clearer. Let me know what you think.

**Review by jbaublitz — APPROVED** [review]

**Pull request merged**